servers.tf

```diff
@@ -19,6 +19,12 @@
 #
 
 # Template for creating Vault nodes
+locals {
+  use_internal_lb = var.load_balancing_scheme == "INTERNAL"
```

Comment:
Should we do something nice like upcase their variable incase they type `internal` instead of `INTERNAL`?